__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+
+beforeEach(() => {
+    return seed(testData);
+  });
+  
+afterAll(() => {
+    return db.end();
+  });
+  
+  describe('get/api/topics', () => {
+    test('responds with an array of topics', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+          expect(Array.isArray(response.body.topics)).toBe(true);
```

Comment:
This would pass for any array at all. We should be checking that each element in the array has the correct keys and the correct amount of entries in the database.